Formula/docksal.rb

```diff
@@ -0,0 +1,25 @@
+class Docksal < Formula
+  desc "Docker powered web development environments for macOS, Windows & Linux"
+  homepage "https://docksal.io"
+  url "https://github.com/docksal/docksal/archive/v1.12.3.tar.gz"
+  sha256 "252a8ce0194fbcacb0a2b7dd6d578843f4ada2cd31c1859350a53791827932a0"
+
+  def install
+    mkdir_p "~/.docksal"
+    touch "~/.docksal/docksal.env"
+    system "echo", "'DOCKER_NATIVE=1'", ">>", "~/.docksal/docksal.env"
+    bin.install "bin/fin"
```

Comment:
So the software is a single shell script?

Reply:
@fxcoudert that is correct. There is similar software like Lando that can be installed via a cask, and they do that so they can install Docker Desktop. Would a better approach here be to also make this a cask even though docksal is a cli tool?